storybook/cypress/e2e/logo.cy.ts

```diff
@@ -0,0 +1,35 @@
+describe("Logo", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:45000/iframe.html?id=core-logo--default");
+  });
+
+  it("shows a label", () => {
+    cy.get("dso-logo")
+      .invoke("attr", "label", "Beheerportaal")
+      .shadow()
+      .find(".logo-content-label")
+      .should("be.visible");
+  });
+
+  it("only shows a label on smaller screens", () => {
```

Comment:
Ik zou deze test ook met `cy.percySnapshot();` uitbreiden, dan maken we van de smalle viewport ook een screenshot.

Sowiezo is die "should be accessible" test 'n beetje een zwaktebod: Eigenlijk willen we dit bij meerdere states van het component testen. Ik zou 'm lekker verwerken in de overige tests. Immers, als een Logo met/zonder label en/of ribbon wordt gemaakt moet ie ook acessibile zijn.